pkg/kubelet/volumemanager/cache/desired_state_of_world.go

```diff
@@ -258,10 +272,16 @@ func (dsw *desiredStateOfWorld) MarkVolumesReportedInUse(
 }
 
 func (dsw *desiredStateOfWorld) DeletePodFromVolume(
-	podName types.UniquePodName, volumeName v1.UniqueVolumeName) {
+	podName types.UniquePodName, volumeName v1.UniqueVolumeName, volumeSpec *volume.Spec) {
 	dsw.Lock()
 	defer dsw.Unlock()
 
+	volumePluginName := "plugin_unknown"
```

Comment:
If a value is unknown the convention is to use `<n/a>`

Reply:
Will update to change the default.